src/components/common/Dot/Dots.module.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../../styles/mediaQueriesMixins.scss";
+@import "../../../styles/settings.scss";
+
+.dots {
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    @include for-tablet {
+        position: relative;
+        margin: 0 auto;
+    }
+
+    @include for-phone {
+        top: 60px;
+    }
+
+    ul {
+        margin: 0;
+        padding: 0 0 0 1rem;
+        list-style: none;
+        background-color: $newFurniture-dots-ul-bg;
+        transform: translateY(15px);
```

Comment:
W tym komponencie nie róbmy takich przesunieć, jeśli jest to konieczne to zróbmy to tak jak opisałem w komentarzu ponizej